CMakeLists.txt

```diff
@@ -35,6 +29,7 @@ option(USE_STLAB_DOUBLECONV "use stlab mirror of double-conversion repository" O
 # with boost 1.60 boost.thread uses boost.move unique_ptr
 # which is broken with BOOST_MOVE_USE_STANDARD_LIBRARY_MOVE flag...
 # fix is to conditionally remove this last boost thread dependency
+# NB: Seems fixed for boost 1.65.1
```

Comment:
not obvious but that is the reason why I can enable all combinations in cmake_build_all.sh